src/widgets/ActivityGroups/ui/ActivityGroupList.tsx

```diff
@@ -47,33 +36,17 @@ type NavigateToActivityDetailsPageProps = {
 export const ActivityGroupList = (props: ActivityListWidgetProps) => {
   const { t } = useCustomTranslation()
   const navigatator = useCustomNavigation()
-  const navigateToActivityDetailsPage = (
-    { appletId, activityId, eventId }: NavigateToActivityDetailsPageProps,
-    options: { isRestart: boolean },
-  ) => {
+  const navigateToActivityDetailsPage = ({ appletId, activityId, eventId }: NavigateToActivityDetailsPageProps) => {
     if (props.isPublic && props.publicAppletKey) {
       return navigatator.navigate(
         ROUTES.publicActivityDetails.navigateTo(appletId, activityId, eventId, props.publicAppletKey),
```

Comment:
Why would we need to pass the public applet key to navigation options? Is it used in the URL?

Reply:
Yes, I need this option in the future in the web application